docs/src/guides/run-ingest-on-terra.rst

```diff
@@ -0,0 +1,79 @@
+**************************
+Run Data Ingest on Terra
```

Comment:
The capitalization here makes it seem like "Data Ingest" is some sort of proper noun within Nextstrain, when that's not the case.

```suggestion
Ingest SARS-CoV-2 data on Terra
```